common/firmware/system/app_update.c

```diff
@@ -26,6 +26,18 @@ void app_update_start() {
     // Disable irqs.
     __disable_irq();
 
+    // systick should be off at boot
+    SysTick->CTRL = 0;
+    SysTick->LOAD = 0;
+    SysTick->VAL = 0;
+
+    /* Clear Interrupt Enable Register & Interrupt Pending Register */
+    for (int i=0;i<8;i++)
```

Comment:
Should we not be doing this in the `SystemInit` function of each `system_stm32l5xx.c` file?